README.md

```diff
@@ -612,31 +612,46 @@ $ git clone https://github.com/encukou/freezeyt
 3. Add your forked repo as a new remote to your local computer:
 
 ```shell
-$ git remote add <name_your_remote> https://github.com/<your_username>/freezeyt
+$ git remote add <remote_label> git@github.com:<username>/freezeyt.git
```

Comment:
And I just checked this with a new container. It looks like you need to have a SSH key set up at GitHub to fetch from a SSH remote.
It's probably best to use `http` in the example.

Reply:
I am not sure but `http` version is not working anymore for push etc. I try to clone with `http` and it seems that only this action is allowed.